doc: add ui section to 6.8 release notes (#18137)
canonical/lxd: doc/reference/release-notes/release-notes-6.8.md

- Applied HTTP timeouts to the pprof, Loki, and endpoint listeners.
- TCP keep alive and TCP user timeout configured on incoming API connections for faster stale connection detection.

## UI updates

This release introduces cluster links, improves instance configuration visibility, and enhances responsiveness across the UI, alongside a range of user-driven fixes and refinements.

### Cluster management

- The UI now supports cluster links for connecting multiple clusters.
- The UI now supports full create, edit, and delete management for cluster member roles.

### Instance experience

> - YAML editor now provides an expanded view that surfaces inherited configuration values (e.g. from profiles) alongside instance-level settings.

Reviewer: ```suggestion
- The YAML editor now provides an expanded view that surfaces inherited configuration values (such as from profiles) alongside instance-level settings.
```
In general, avoid Latinisms as they're often misunderstood, especially in international contexts.

- More responsive instance creation with live progress updates via events.
- Ubuntu-themed terminal for instances.

### Forms and input enhancements

- Introduced prefixed inputs for IP address assignments for clearer networking configuration.
- Added output fields to forms where applicable.

### Asynchronous operations

- Improved handling of long-running operations with asynchronous support for:
    - Networks
    - Network peerings
    - Network ACLs
    - Storage pools
    - Storage buckets and bucket keys

### User-driven improvements

- Instances uploaded from file now appear immediately in the instance list while processing.
- Improved visibility of available storage pool size.

> - Enhanced storage driver selection with more detailed information.

Reviewer: ```suggestion
- Enhanced storage driver selection with more details.
```
